src/seedu/addressbook/AddressBook.java

```diff
@@ -394,10 +394,22 @@ private static String executeCommand(String userInputString) {
      * @return  size 2 array; first element is the command type and second element is the arguments string
      */
     private static String[] splitCommandWordAndArgs(String rawUserInput) {
-        final String[] split =  rawUserInput.trim().split("\\s+", 2);
-        return split.length == 2 ? split : new String[] { split[0] , "" }; // else case: no parameters
+        final String[] splitCommandWordandArgs =  trimRawUserInput(rawUserInput);
+        if(splitCommandWordandArgs.length == 2) {
+        	return splitCommandWordandArgs;
```

Comment:
Notice that your indentation mechanism uses tabs for indentation. 
The coding standards we follow require using 4 spaces for indentation.
Tip: you can set Eclipse to do this for you, check out how!